src/main/resources/db/changelog/changelog.sql

```diff
@@ -928,3 +928,8 @@ ADD CONSTRAINT FOREIGN KEY (`sessionGuid`) REFERENCES `Sessions` (`guid`) ON DEL
 
 ALTER TABLE `Substudies`
 ADD COLUMN `studyStartEventId` varchar(255);
+
+-- changeset bridge:64
+
+ALTER TABLE `WeeklyAdherenceReports`
+ADD COLUMN `week` int(4);
```

Comment:
Could call is weekInStudy or studyWeek....

Reply:
Updated to WeekInStudy here and in other places where week is being reported, but not EventStreamDay because the week value there is only set for the event stream report.